lib/matplotlib/backend_bases.py

```diff
@@ -2559,6 +2674,120 @@ class NonGuiException(Exception):
     pass
 
 
+class WindowEvent(object):
+    def __init__(self, name, window):
+        self.name = name
+        self.window = window
+
+
+class WindowBase(cbook.EventEmitter):
+    """The base class to show a window on screen.
+
+    Parameters
+    ----------
+    title : str
+        The title of the window.
+    """
+
+    def __init__(self, title, **kwargs):
+        super(WindowBase, self).__init__(**kwargs)
```

Comment:
Yes, I did have the same thought last night (as I typed the message), that I probably should refactor that down now we use super...

But now checking on the other backends, and I now see I call ``self.set_window_title(title)`` in all the backends apart from Wx, and I don't think it hurts to call it again, so I can probably move that to here :).  Fingers crossed the super magic holds (I think it will, but only 90% sure)...